src/main/kotlin/no/nav/familie/ef/sak/mapper/PersonopplysningerMapper.kt

```diff
@@ -13,39 +16,51 @@ class PersonopplysningerMapper(private val adresseMapper: AdresseMapper,
-                                                     gyldigTilOgMed = it.gyldigTilOgMed,
-                                                     motpartsPersonident = it.motpartsPersonident,
-                                                     navn = identNavn[it.motpartsPersonident])
-                                     },
-                                     egenAnsatt = egenAnsatt,
-                                     navEnhet = arbeidsfordelingService.hentNavEnhet(ident)
-                                                        ?.let { it.enhetId + " - " + it.enhetNavn } ?: "Ikke funnet"
-        )
+        val søker = personMedRelasjoner.søker
+        return PersonopplysningerDto(
+                adressebeskyttelse = søker.adressebeskyttelse.firstOrNull()
+                        ?.let { Adressebeskyttelse.valueOf(it.gradering.name) },
+                folkeregisterpersonstatus = søker.folkeregisterpersonstatus.firstOrNull()
+                        ?.let { Folkeregisterpersonstatus.fraPdl(it) },
+                dødsdato = søker.dødsfall.firstOrNull()?.dødsdato,
+                navn = NavnDto.fraNavn(søker.navn.gjeldende()),
+                kjønn = søker.kjønn.single().kjønn.let { Kjønn.valueOf(it.name) },
+                personIdent = ident,
+                telefonnummer = søker.telefonnummer.find { it.prioritet == 1 }
+                        ?.let { TelefonnummerDto(it.landskode, it.nummer) },
+                statsborgerskap = statsborgerskapMapper.map(søker.statsborgerskap),
+                sivilstand = søker.sivilstand.map {
+                    SivilstandDto(type = Sivilstandstype.valueOf(it.type.name),
+                                  gyldigFraOgMed = it.gyldigFraOgMed?.toString() ?: it.bekreftelsesdato,
+                                  relatertVedSivilstand = it.relatertVedSivilstand,
+                                  navn = identNavn[it.relatertVedSivilstand])
+                },
+                adresse = tilAdresser(søker),
+                fullmakt = fullmakter.map {
+                    FullmaktDto(gyldigFraOgMed = it.gyldigFraOgMed,
+                                gyldigTilOgMed = it.gyldigTilOgMed,
+                                motpartsPersonident = it.motpartsPersonident,
+                                navn = identNavn[it.motpartsPersonident])
+                },
+                egenAnsatt = egenAnsatt,
+                navEnhet = arbeidsfordelingService.hentNavEnhet(ident)
+                                   ?.let { it.enhetId + " - " + it.enhetNavn } ?: "Ikke funnet",
+                barn = personMedRelasjoner.barn.map {
+                    mapBarn(
+                            it.key,
+                            it.value,
+                            personMedRelasjoner.søkerIdent,
+                            personMedRelasjoner.søker.bostedsadresse,
+                            identNavn)
+                },
+                innflyttingTilNorge = søker.innflyttingTilNorge,
```

Comment:
Innflytting/Utflytting har folkeregistermetadata som er unødvendig å sende med till frontend, så tenker det er fint å mappe om den til egne DTO'er

Reply:
I frontend bruker vi folkeregistermetadata til å vise dato for innflytting/utflytting. Men det er kanskje feil bruk av data'en?

Reply:
Ah ok, vet ikke om det er helt riktig dato, https://navikt.github.io/pdl/#_innflytting 
"Gyldighetstidspunktet fra Freg på innflytting er likt ajourholdstidspunktet og har ingen sammenheng med når innflyttingen skjedde i virkeligheten." 
"Dersom man skal finne ut når en innflytting gjelder fra, så må man se på bostedsadressens dato i feltet angitt flyttedato."

Syns denne er litt vanskelig, vi kan gi ut disse datoene, men vet ikke hva det gir av verdi :/ Hadde vært fint hvis PDL hadde kunnet gi oss litt bedre data for dette. 

Husk att naming av felter blir litt annerledes enn det vi har på andre felter då pdl ikke bruker åæø i json.
Vet